### PR TITLE
0.9 port, docs

### PR DESCRIPTION
**Pull request by atotic**
Port to 0.9, slight update of documentation

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,22 +1,15 @@
...
-This code may only be used under the BSD style license found at https://polymer.github.io/LICENSE
-The complete set of authors may be found at https://polymer.github.io/AUTHORS
-The complete set of contributors may be found at https://polymer.github.io/CONTRIBUTORS
-Code distributed by Google as part of the polymer project is also
-subject to an additional IP rights grant found at https://polymer.github.io/PATENTS
--->
+<!-- Copyright (c) 2015 Google Inc. All rights reserved. -->
 <html>
 <head>
-  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1.0, user-scalable=yes">
```
Comment (philipwalton) [inline index.html]: Why is this line going away?

Comment (atotic) [inline index.html]: Just boilerplate we've been using. Happy to put it back in.

Comment (philipwalton) [inline index.html]: I don't now if something changed for the whole GWC project, but I noticed it wasn't removed in the [`<google-sheets>` PR](https://github.com/GoogleWebComponents/google-sheets/pull/18/), so I wanted to ask about it.

Comment (ebidel) [inline index.html]: Let's keep it. It's good for mobile

**Comment by atotic** [comment]
meta is back

**Comment by philipwalton** [comment]
LGTM

**Pull request merged**